### PR TITLE
Feature/initial component structure

### LINKED ISSUE #2: React: Component File Structure
As a developer, 
I should be able to access and edit files representing each component 
rendered within the SilverNest app.

### LINKED ISSUE #3: Component: About
As a user, 
when I open up the app, I should land on the About view, 
which includes images and a description of SilverNest,
so that I know what kind of site I am visiting.

### LINKED ISSUE #4: Component: Thumbnails
As a dev,
I should have a basic grid list design with dummy data 
including image and text,
so that I can later plug in data from the database to display available residents.

### LINKED ISSUE #5: React Router
As a dev, 
I should have React BrowserRouter library installed in the project, 
and basic implementation in the project
so that I can see url paths corresponding with the "page" displayed.

### PR DESCRIPTION
**Pull request by lo-la-do-li**
### What’s this PR do?
The SilverNest App now has component folders, files, and some basic starter UI, including:
- React Browser Router for /about and /get-involved
- Material UI Components for NavBar, Thumbnails (scrolling GridList), and About (image GridList)

The following libraries/dependencies have been installed:
- node-sass (version 4.14.1)
- react-router-dom
- material-ui/core
- CssBaseline from material-ui/core
### Where should the reviewer start?
- App, About, NavBar, Thumbnails line 1
- package.json
- index.js for BrowserRouter and CssBaseline

### How should this be manually tested?
- Run npm start, switch between About and Get Involved tabs to see changes reflected in URL
- View scrollable grid list

### Any background context you want to provide?
N/A

### Screenshots (if appropriate):
About route/page
Get Involved route/page

### Questions:
- How should we render resident thumbnails in Get Involved? A modal, conditional rendering once semester is selected and submitted?
- When/where should we make our API call to retrieve residents data? I'm thinking onSubmit in the Form (once we build out this component)

### CODE REVIEW

File: src/About/About.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
```
Comment (cameronRomo) [inline src/About/About.js:2]: Super interesting, I love the way you can bring in various components.

File: src/App/App.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  return (
+
+    <div className="App">
+      <Box component="span" m={1}>
+        <header>
+          <h1>Silver Nest</h1>
+        </header>
+      </Box>
+      <NavBar />
+      <Switch>
```
Comment (cameronRomo) [inline src/App/App.js:19]: Nice work getting the router set up.

File: src/Thumbnails/Thumbnails.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+import GridList from '@material-ui/core/GridList';
+import GridListTile from '@material-ui/core/GridListTile';
+import GridListTileBar from '@material-ui/core/GridListTileBar';
+import IconButton from '@material-ui/core/IconButton';
+
+// import StarBorderIcon from '@material-ui/icons/StarBorder';
+
+
+const useStyles = makeStyles((theme) => ({
+  root: {
```
Comment (cameronRomo) [inline src/Thumbnails/Thumbnails.js:12]: Styling with objects? This styling is really interesting.

**Review by cameronRomo — APPROVED** [review]
So far, material-ui is super interesting. I really like how you can bring in various components, and link them with the router, as well as style them on the spot.

File: src/About/About.js (hunk 1 above)
Comment (chadrick-d-dev) [reply src/About/About.js:2]: Material UI is looking super cool!

File: src/NavBar/NavBar.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import { Link } from 'react-router-dom';
+
+const useStyles = makeStyles({
+  root: {
+    flexGrow: 1,
```
Comment (chadrick-d-dev) [inline src/NavBar/NavBar.js:10]: This is css styling?

**Review by chadrick-d-dev — APPROVED** [review]
Way to go! All these additions are great! I love all your choices with the specific material ui elements.

**Pull request merged**